### PR TITLE
Fix typo in MacOS part building.md

### PR DESCRIPTION
**Pull request by CreativeQuantum**
``` 
brew install \
        python@3 \ <--- Forgot the antislash here
        llvm \
        eigen \
        opencascade \
        pkg-config \
        gtk4 \
        gtkmm4 \
        glm \
        range-v3 \
        mimalloc \
        pygobject3 \
        librsvg
```

### CODE REVIEW

File: doc/building.md [hunk 1]
```diff
@@ -148,4 +148,4 @@ brew install --HEAD meson
 
 Now you can build the project using the `./scripts/build_macos.sh` script.
 
-You should now have the `dune3d` executable in the `build/` folder.
```
Comment (CreativeQuantum) [inline doc/building.md:151]: Not sure why this is highlighted.

**Comment by CreativeQuantum** [comment]
Had to `pip3 install pycairo` too.

**Comment by vespakoen** [comment]
@CreativeQuantum those changes seem good to me, can you add[py3cairo](https://formulae.brew.sh/formula/py3cairo) to the homebrew deps as well?

It would also be good to include a note that the macOS version is not really working at the moment (selection is broken because of the 1px line width limitation on macOS).

File: doc/building.md (hunk 1 above)
Comment (vespakoen) [reply doc/building.md:151]: I just wanted to point out where the binary can be found, because macOS users might expect a .app folder or something.

**Comment by CreativeQuantum** [comment]
Not necessary it seems.

```
brew deps pygobject3
[...]
py3cairo
[...]
```

Another way to fix this error:
```
File "/Users/creativequantum/Documents/Code/GitHub/dune3d/build/../make_icon_texture_atlas.py", line 5, in <module>
    import cairo
ModuleNotFoundError: No module named 'cairo'
```

```sh
pip3 uninstall pycairo # (because I installed it as explained above)
brew remove --ignore-dependencies pygobject3 py3cairo cairo
brew install pygobject3 py3cairo cairo
```

**Pull request merged**